iotdb-core/antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4

```diff
@@ -1141,6 +1141,22 @@ loadFileAttributeClause
     | ONSUCCESS operator_eq (DELETE|NONE)
     ;
 
+// Load TsFile With
+loadFileWith
+    : LOAD fileName=STRING_LITERAL loadFileWithAttributeClauses
+    ;
+
+loadFileWithAttributeClauses
+    : WITH
+        LR_BRACKET
+        (loadFileWithAttributeClause COMMA)* loadFileWithAttributeClause?
+        RR_BRACKET
+    ;
+
+loadFileWithAttributeClause
```

Comment:
I think those attributes in pipes can be unified.. Because there are so many "string = string"s in antlr

Reply:
I think so, too

Reply:
I think so, too